src/components/projects-summary/projects-summary.html

```diff
@@ -72,15 +69,24 @@ <h4 class="catalog-modal-title">
       </div>
     </div>
   </div>
-  <h2 class="secondary">Resources</h2>
-  <span>
-    {{$ctrl.resourcesText}}
-  </span>
-  <div class="resources-panel">
-    <ul>
-      <li ng-repeat="resource in $ctrl.resources">
-        {{resource.text}}
-      </li>
-    </ul>
+  <div ng-if="$ctrl.showGetStarted">
+    <div class="getting-started-panel">
+      <h2 class="secondary" ng-if="$ctrl.projects && $ctrl.projects.length > 0">Getting Started</h2>
+      <button class="getting-started-button btn btn-default" ng-class="{'with-projects': $ctrl.projects && $ctrl.projects.length > 0}" ng-click="$ctrl.handleGettingStartedClick()">
+        <span class="fa fa-info-circle fa-2"></span>
+        <span class="getting-started-button-text">Start Guided Tour</span>
+      </button>
+    </div>
+    <div class="resources-panel">
+      <h2 class="secondary">Resources</h2>
+      <p>
+        {{$ctrl.resourceDescription}}
+      </p>
+      <ul>
+        <li ng-repeat="resource in $ctrl.resourceLinks">
+          <a href="" ng-click="$ctrl.onResourceClick(resource)">{{resource.title}}</a>
```

Comment:
We're using this pattern a lot for links, but it breaks things like right-click -> open new window / copy link. It also prevents being able to hover over the link and see the URL. Is there a way we can set a real href?

I have the same concern for the project links, which are all callbacks.